diktat-maven-plugin/pom.xml

```diff
@@ -48,9 +49,15 @@
             <version>${ktlint.version}</version>
         </dependency>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.13.1</version>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
```

Comment:
why vintage engine?

Reply:
`maven-testing-harness`, maven library for unit testing, supports only junit 3 and 4